src/components/Cards/AllCards.jsx

```diff
@@ -6,8 +6,7 @@ export class AllCards extends React.Component {
     getCardItems() {
         if (this.props.cards) {
             return this.props.cards.map(card => (
-                <img className={styles.allCards} key={card.id} src={require(`../Dashboard/dixit_logo.png`)}></img>
-                
+                <img className={styles.allCards} key={card.id} src={require(`../Dashboard/dixit_logo.png`)}></img>     
```

Comment:
Please can you add an alt here of the card id like in the other places just so React doesn't complain